spec/models/results/response_tree_builder_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "response tree" do
```

Comment:
For model specs like this you should do `describe Results::ResponseTreeBuilder`

Reply:
done